### DISCUSSION TITLE
crash on old file name

### DISCUSSION THREAD

**Discussion by hamaryns** [beetbox/beets#5490] [Q&A]
Hi,

I recently renamed the directory with ‘music to be sorted’.  Now if I release beets on it, it crashes on a song with a FileNotFound because it is looking for the song in the old name of the directory.  How can I make beets forget this old directory?

[beet12.log](https://github.com/user-attachments/files/17563133/beet12.log)

TIA, H.

**Comment by snejus** [comment]
Have you updated the `directory` field in your beets config with your new directory name?

**Reply by hamaryns** [reply]
That is not relevant, it is not the music library directory which has been renamed, but the directory to which I point it on the command line with music to be sorted.
